templates/detailItems/url.html

```diff
@@ -6,4 +6,4 @@
 	   focus-expression="$parent.ctrl.focus === ctrl.name"
 />
 <button ng-click="ctrl.deleteField()" class="icon-delete" title="Delete"></button>
-<a class="url-link" target="_blank" href="{{ctrl.data.value}}">↗</a>
+<a ng-if="ctrl.data.value.length" class="url-link" target="_blank" href="{{ctrl.data.value}}">↗</a>
```

Comment:
I experimented a bit on this. Seems like the pattern angular uses requires `http://` or `https://`.